### PR TITLE
create formula flags

### PR DESCRIPTION
**Pull request by maurineimirandazup**
Signed-off-by: maurineimirandazup <maurinei.miranda@zup.com.br>

### CODE REVIEW

File: content/pt-br/Standard Inputs/Comandos Core.md [hunk 1]
```diff
@@ -25,7 +25,15 @@ rit init
 rit init --sendMetrics="yes" --addCommons="yes" --runType="local"
 ```
 
-### Comandos de credenciais
+### Comando de criar formula
```
Comment (jessica-tw) [inline content/pt-br/Standard Inputs/Comandos Core.md:28]:
```suggestion
### Comando para criar fórmula
```

File: content/pt-br/Standard Inputs/Comandos Core.md [hunk 2]
```diff
@@ -25,7 +25,15 @@ rit init
...
+
+rit create formula
+
+```text
+rit create formula --name="rit group verb noun" --language="go" --workspace="Default"
+```
+
+### Comandos de Credenciais
```
Comment (jessica-tw) [inline content/pt-br/Standard Inputs/Comandos Core.md:36]:
```suggestion
### Comandos de credenciais
```

File: content/en/Standard Inputs/Core Commands.md [hunk 3]
```diff
@@ -25,6 +25,14 @@ rit init
...
 
+### Create formula command
+
+rit create formula
+
+```text
+rit create formula --name="rit group verb noun" --language="go" --workspace="Default"
+```
+
 ### Credentials commands
```
Comment (jessica-tw) [inline content/en/Standard Inputs/Core Commands.md:36]:
```suggestion
### Credentials command
```

**Review by jessica-tw — COMMENTED** [review]
TW review: done ;)

**Review by GuillaumeFalourd — APPROVED** [review]

**Pull request merged**